infrastructure/code/code_test.go

```diff
@@ -369,8 +370,8 @@ func TestUploadAndAnalyze(t *testing.T) {
 				t, ux2.AnalysisIsReadyProperties{
 					AnalysisType:      ux2.CodeSecurity,
 					Result:            ux2.Success,
-					FileCount:         metrics.lastScanFileCount,
-					DurationInSeconds: metrics.lastScanDurationInSeconds,
+					FileCount:         1,
```

Comment:
`metrics` is not a pointer anymore so it wouldn't have the updated value for `lastScanFileCount`. I think it makes more sense to test the values directly here but let me know if there was some other reason why we might want `metrics` here to be used